remix_01.py

```diff
@@ -3,12 +3,15 @@
 
 def main():
 
+    # Read the json file as json
     filename = "remixes_jan2016_dec2016.json"
-    file  = open(filename, "r")
-    for line in file:
-        if "file_page_url" in line:
-            id = line.split('/')[-1][:-3]
-            filter_json(id)
+    with open(filename, "r") as infile:
```

Comment:
Before we can solve this, we changed the loading of the initial json list (the list of remixes from last year), into something a bit cleaner (and using the json module to read the data)

See this line and the following